### PR TITLE
Surf gear in boathouse page

### PR DESCRIPTION
**Pull request by mprat**
@tarynosaurus here is one way to combine the info!

### CODE REVIEW

**Comment by mprat** [comment]
If one of the webmaster @fpagnoux or @DavidCain could give me a :+1: on the jekyll that would be sweet. Taryn will give me the OK on the content before this gets merged

**Comment by mprat** [comment]
(replaces https://github.com/mitoc/website/pull/125)

File: _pages/rentals/boathouse.md [hunk 1]
```diff
@@ -129,6 +123,13 @@ If you've never done this before, read on for...
     </tr>
 </tbody></table>
 
+### Surfing
+
+Approver is the Windsurf chair for the appropriate gear ([boathouse-mgr@mit.edu](mailto:boathouse-mgr@mit.edu))
```
Comment (fpagnoux) [inline _pages/rentals/boathouse.md:128]:
```suggestion
Approver is the Surf chair for the appropriate gear ([boathouse-mgr@mit.edu](mailto:boathouse-mgr@mit.edu))
```

File: _includes/price-table.html [hunk 2]
```diff
@@ -1,4 +1,9 @@
 {% assign category = include.category %}
+{% assign approver = mitoc-approval@mit.edu %}
```
Comment (fpagnoux) [inline _includes/price-table.html:2]: We might need quotes here? Maybe that's why the assignment doesn't seem to be working?

**Review by fpagnoux — APPROVED** [review]
Looks good to me, just need to fix the CI issue

**Comment by mprat** [comment]
Yes I think it's the quotes. I fell asleep thinking this but didn't get a
chance to test. Will fix today

File: _pages/rentals/boathouse.md (hunk 1 above)
Comment (mprat) [reply _pages/rentals/boathouse.md]: edited this in the latest commit

**Pull request merged**